client/dashboard/plugins/scheduled-updates/new/index.tsx

```diff
@@ -2,7 +2,9 @@ import {
 	updateSchedulesBatchCreateMutation,
 	siteJetpackMonitorSettingsCreateMutation,
 } from '@automattic/api-queries';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
```

Comment:
Please use `useAnalytics()` from 

https://github.com/Automattic/wp-calypso/blob/f128ec92b987b92950ec1d6cd8f5ebeb353fc020/client/dashboard/app/analytics/index.tsx#L22

I think currently in the codebase there are many wrong instances; I'll try to clean them up.